.github/workflows/main.yml

```diff
@@ -49,8 +49,32 @@ jobs:
       uses: actions/setup-dotnet@v2
       with:
         dotnet-version: ${{ env.DOTNET_VERSION }}
-    - name: Pack with dotnet
-      run: git rev-list --count HEAD |xargs printf "CI%05d" |xargs dotnet pack -c Debug -o out --include-source --version-suffix
+    - name: Set Version
+      run: git rev-list --count HEAD | xargs printf 'CI%05d' | xargs -I{} echo 'VERSION={}' >> $GITHUB_ENV
+    - name : Pack (Neo)
+      run: |
+        dotnet pack ./src/Neo \
+        --configuration Debug \
```

Comment:
--include-source ?

Reply:
what does that do? Symbols? if so they get automatically generated on published.

Reply:
```
--include-symbols - it creates the symbols package.
--include-source - it creates the symbols package with a src folder inside containing the source files.
```

Reply:
Oh ya i we can do that.

Reply:
that won't work see https://github.com/NuGet/Home/issues/8589, spec doesnt allow it anymore, cause of the new format.